spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -153,3 +153,20 @@ The outbound channel adapter maps every header from Spring messages into Google
 In the process, the outbound mapper also converts the value of the headers into string.
 
 Each adapter declares a `setHeaderMapper()` method to let you further customize which headers you want to map from Spring to Google Cloud Pub/Sub, and vice-versa.
+
+For example, to filter out headers `"foo"`, `"bar"` and all headers starting with the prefix "prefix_", you can use `setHeaderMapper()` along with the `PubSubHeaderMapper` implementation provided by this module.
+
+[source,java]
+----
+PubSubInboundChannelAdapter adapter = ...
+...
+PubSubHeaderMapper headerMapper = new PubSubHeaderMapper();
+headerMapper.setOutboundHeaderPatterns("!foo", "!bar", "!prefix_*", "*");
+adapter.setHeaderMapper(headerMapper);
+----
+
+NOTE: The order in which the patterns are declared in `PubSubHeaderMapper.setOutboundHeaderPatterns()` and `PubSubHeaderMapper.setInboundHeaderPatterns()` matters.
+The first patterns have precedence over the following ones.
+
+In the previous example, the `"*"` pattern means every header is mapped.
+However, because it comes last in the list, the previous patterns take precedence.
```

Comment:
You may send end-user from here to JavaDocs of the `org.springframework.integration.support.utils.PatternMatchUtils.smartMatch()`.